Presentation.Web/App_Start/MappingConfig.cs

```diff
@@ -74,10 +74,6 @@ protected override void Configure()
                   .ReverseMap()
                   .ForMember(dest => dest.References, opt => opt.Ignore());
 
-            Mapper.CreateMap<ItInterfaceType, OptionDTO>()
-                  .ReverseMap()
-                  .ForMember(dest => dest.References, opt => opt.Ignore());
-
             Mapper.CreateMap<InterfaceType, OptionDTO>()
```

Comment:
Hvad er forskellen på ItInterfaceType og InterfaceType??

Reply:
InterfaceType definere et IT system type